### PR TITLE
The preview of weg was not aligned well

### PR DESCRIPTION
**Pull request by luriusTM**
Only placement of the position has been calculated based on the configuration value.

### CODE REVIEW

**Comment by eythaann** [comment]
@luriusTM thanks for you fix, this will be merged but first you need to sign the commit in order to be verified.
You can read more about this: https://docs.github.com/en/authentication/managing-commit-signature-verification/about-commit-signature-verification

**Review by eythaann — CHANGES_REQUESTED** [review]

File: src/apps/seelenweg/modules/item/infra/UserApplication.tsx [hunk 1]
```diff
@@ -30,8 +32,20 @@ export const UserApplication = memo(({ item }: Props) => {
   );
 
   const [openPreview, setOpenPreview] = useState(false);
+  const settings = useSelector(Selectors.settings);
 
   const { t } = useTranslation();
+  const calculatePlacement = (position: any) => {
+    if (position == SeelenWegSide.Bottom) {
```
[the remark this answers is not in the document]
Comment (luriusTM) [reply src/apps/seelenweg/modules/item/infra/UserApplication.tsx]: code technically you are right, although linter only allows me this way to commit a switch, so it is not look better. Inline switch is not available in js.

[the remark this answers is not in the document]
Comment (eythaann) [reply src/apps/seelenweg/modules/item/infra/UserApplication.tsx]: linter allows switch, only made it line by line instead all inline.

File: src/apps/seelenweg/modules/item/infra/UserApplication.tsx [hunk 2]
```diff
@@ -36,14 +36,22 @@ export const UserApplication = memo(({ item }: Props) => {
...
+      case SeelenWegSide.Top: {
+        return 'bottom';
+      }
+      case SeelenWegSide.Left: {
+        return 'right';
+      }
+      case SeelenWegSide.Right: {
+        return 'left';
+      }
+      default: {
```
Comment (eythaann) [inline src/apps/seelenweg/modules/item/infra/UserApplication.tsx:52]: default should through an exception of type "Unimplemented".

**Review by eythaann — CHANGES_REQUESTED** [review]

**Review by eythaann — APPROVED** [review]

**Pull request merged**